lib/spack/spack/ci.py

```diff
@@ -705,14 +722,51 @@ def generate_gitlab_ci_yaml(
             files (spack.yaml, spack.lock), etc should be written.  GitLab
             requires this to be within the project directory.
     """
+    rev1, rev2 = get_change_revisions()
+    tty.warn(f"Got following revisions: rev1={rev1}, rev2={rev2}")
+
+    # Get the joined "ci" config with all of the current scopes resolved
+    ci_config = cfg.get("ci")
+    spack_prune_untouched = os.environ.get("SPACK_PRUNE_UNTOUCHED", None)
+
+    changed = rev1 and rev2
+    affected_pkgs = None
+    if spack_prune_untouched and changed:
+        affected_pkgs = compute_affected_packages(rev1, rev2)
+        tty.debug("affected pkgs:")
+        for p in affected_pkgs:
+            tty.debug(f"  {p}")
+
+        possible_builds = spack.package_base.possible_dependencies(*env.user_specs)
+        # with env.read_transaction():
```

Comment:
What's the purpose of this comment?

Reply:
from the old method, missed it when I deleted the other parts.